fe/fe-core/src/main/java/com/starrocks/transaction/PublishVersionDaemon.java

```diff
@@ -564,7 +566,13 @@ private void deleteTxnLogIgnoreError(ComputeNode node, DeleteTxnLogRequest reque
             LakeService lakeService = BrpcProxy.getLakeService(node.getHost(), node.getBrpcPort());
             // just ignore the response, for we don't care the result of delete txn log
             // and vacuum will clan the txn log finally if it failed.
-            lakeService.deleteTxnLog(request);
+            // make sure sync wait for the RPC call to avoid too many RPC call existed in BE/CN side
+            Future<DeleteTxnLogResponse> responseFuture = lakeService.deleteTxnLog(request);
+            try {
+                DeleteTxnLogResponse response = responseFuture.get();
+            } catch (Exception e) {
+                LOG.warn("delete txn log error: " + e.getMessage());
+            }
```

Comment:
```suggestion
            DeleteTxnLogResponse response = responseFuture.get();
            if (response.xxx) ...
```

Reply:
Maybe we need not check the error status here? This RPC is called in `deleteTxnLogIgnoreError ` which means we don't care the result whether the rpc call is success or not

Reply:
it is better to add error log when delete rpc is failed.
and the inner try catch is not necessary.

Reply:
updated